build-tools/automation/yaml-templates/variables.yaml

```diff
@@ -0,0 +1,33 @@
+variables:
+- name: RunningOnCI
+  value: true
+- name: XA.Build.Configuration
+  value: Release
+- name: XA.Jdk8.Folder
+  value: jdk-1.8
+- name: XA.Jdk11.Folder
+  value: jdk-11
+- name: NuGetArtifactName
+  value: nuget-unsigned
+- name: InstallerArtifactName
+  value: installers-unsigned
+- name: TestAssembliesArtifactName
+  value: test-assemblies
+- name: NUnitConsoleVersion
+  value: 3.11.1
+- name: NUnit.NumberOfTestWorkers
+  value: 4
+- name: DotNet6Version
+  value: 6.0.100-rc.2.21505.57
```

Comment:
If we are using public builds now, could we change `use-dot-net.yaml` to use the built-in `UseDotNet` task again?

This would be for a separate PR.